spec/unit/provider/ifconfig_spec.rb

```diff
@@ -16,7 +16,7 @@
 # limitations under the License.
 #
 
-# require File.expand_path(File.join(File.dirname(__FILE__), "..", "..", "spec_helper"))
+# require "spec_helper"
```

Comment:
this should just get removed

Reply:
I did remove this at one point but kept `git reset`ing because I was messing up my `sed` commands. :)